src/common/mclock_common.cc

```diff
@@ -490,11 +490,10 @@ void MclockConfig::mclock_handle_conf_change(const ConfigProxy& conf,
               "\"who\": \"" + osd + "\", "
               "\"name\": \"" + *key + "\""
             "}";
-          std::vector<std::string> vcmd{cmd};
 
           dout(10) << __func__ << " Removing Key: " << *key
                    << " for " << osd << " from Mon db" << dendl;
-                   monc->start_mon_command(vcmd, {}, nullptr, nullptr, nullptr);
+	  monc->start_mon_command({std::move(cmd)}, {}, nullptr, nullptr, nullptr);
```

Comment:
Aren't you changing more than the value category here (i.e. by moving from a vector to a single string)?

Reply:
(value category: lvalue, rvalue, etc.)
I was ref'ing the change that replaced the vector of strings with one string (which seemed to be unrelated)

Reply:
> that replaced the vector of strings with one string (which seemed to be unrelated)

I did not replace a vector or strings with one string. Maybe you missed the curly braces?

Reply:
> I did not replace a vector or strings with one string. Maybe you missed the curly braces?

Yes - I did miss that, but now I wonder what will happen there. Going to check.

Reply:
> Are you sure there are no cases where the client did use the argument?

I checked all changes (by code review, not by running them - I don't have any Ceph installation that I can use to test this), and I'm confident that they are correct. Minus human mistakes, as always.